packages/host/app/components/operator-mode/container.gts

```diff
@@ -590,9 +607,60 @@ export default class OperatorModeContainer extends Component<Signature> {
   };
 
   @action updateSubmode(submode: Submode) {
+    switch (submode) {
+      case Submode.Interact:
+        this.onCodePathChange(null);
+        break;
+      case Submode.Code:
+        this.onCodePathChange(new URL(this.lastCardInRightMostStack.id));
+        break;
+    }
```

Comment:
same comment as above--add an assertNever for the default case so types will should us what to fix if we add a new submode